### PR TITLE
Added the Chrono Miner traits

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/ra2/issues/134.

### CODE REVIEW

File: OpenRA.Mods.RA2/Traits/ChronoResourceDelivery.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	public class ChronoResourceDeliveryInfo : ITraitInfo, Requires<HarvesterInfo>
+	{
+		[Desc("The number of ticks between each check to see if we can teleport to the refinery.")]
+		public readonly int CheckTeleportDelay = 10;
+
+		[Desc("Image used for the teleport effects. Defaults to the actor's type.")]
+		public readonly string Image = null;
+
+		[Desc("Sequence used for the effect played where the harvester jumped from.")]
+		[SequenceReference("Image")] public readonly string WarpInSequence = "warpin";
```
Comment (GraionDilach) [inline OpenRA.Mods.RA2/Traits/ChronoResourceDelivery.cs]: These defaults are wrong.

**Comment by GraionDilach** [comment]
```
ChronoResourceDelivery:
		ChronoInSequence:
		ChronoOutSequence: warpout
		ChronoInSound: vchrtele.wav
		ChronoOutSound: vchrtele.wav
```

was how RA2 set it up. warpin is unused.

**Comment by Mailaender** [comment]
Updated.

**Comment by GraionDilach** [comment]
:+1:

File: OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj [hunk 2]
```diff
@@ -43,6 +43,13 @@
     <Reference Include="OpenRA.Mods.TS">
       <HintPath>dependencies\OpenRA.Mods.TS.dll</HintPath>
     </Reference>
+    <Reference Include="Eluant">
+      <HintPath>dependencies\OpenRA.Mods.TS.dll</HintPath>
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj]: This doesn't look correct.

Comment (Mailaender) [inline OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj]: Fixed.

**Comment by Iamgoofball** [comment]
warpin IS used but not for chrono miners

i think legionaiires use it

**Comment by abcdefg30** [comment]
Looks good to me (apart from a small polish issue). :+1: / :white_check_mark: 
Thanks for reviving this!

**Pull request merged**